library/fedora

```diff
@@ -30,10 +30,10 @@ arm32v7-Directory: armhfp/
 arm64v8-Directory: aarch64/
 ppc64le-Directory: ppc64le/
 
-Tags: branched, 28
+Tags: latest, 28
 Architectures: amd64, arm32v7, arm64v8, s390x
```

Comment:
Doh, I think this isn't actually accurate anymore: https://github.com/fedora-cloud/docker-brew-fedora/tree/0ef99659611684364cda45192c3558fc7bcbc2d5

(Should probably be `Architectures: amd64, arm32v7, arm64v8, ppc64le` instead, with updated `Directory` entries below.)

Reply:
https://doi-janky.infosiftr.net/job/multiarch/view/images/view/fedora/ seems to agree :sweat_smile: